ha/core/event_manager/action_event.py

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python3
```

Comment:
Can we move this file to ha/core/event_manager/model/ directory?
@ajaysrivas

Reply:
yes. That would be better.

Reply:
Updated